src/components/Dashboard/TodoList.vue

```diff
@@ -442,6 +444,7 @@ export default {
           case FilingTypes.ANNUAL_REPORT: {
             const ARFilingYear = todo.header.ARFilingYear
             this.taskItems.push({
+              id: -1, // not falsy
```

Comment:
This is needed so that the conditional on line 146 isn't "true" during unit tests. (Because "inProcessFiling" prop defaults to null and "id" was previously undefined. Those should not be strictly equal but when Jest is running, apparently they are.)